src/transformStateWithClones.js

```diff
@@ -7,7 +7,47 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const arrayObjects = [];
+  let currentState = {};
```

Comment:
```suggestion
  let currentState = { ...state };
```
Объяви переменную сразу как копию state